extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -397,6 +397,8 @@ export class AmpAndroidAppBanner extends AbstractAppBanner {
  */
 function handleDismiss(state) {
   hideBanner(state);
+  // Restore focus to document after dismiss for screen readers.
+  state.element.ownerDocument.body./*REVIEW*/focus();
```

Comment:
true we don't want that in all cases, but I think `removeFromFixedLayer` can still handle this situation so we don't run into this bug again. I think this issue also exists for sidebar, user-notification and sticky-ad IF they are closed/dismissed while focus is inside them, which is not unlikely. 

What do you think about this:
inside `removeFromFixedLayer`, if `document.activeElement` is a child of `element`, then move the focus on body.